### ISSUE TITLE
Cucumber 6

### ISSUE THREAD

**Issue by john-caleb-phillips** [trivago/cucable-plugin#172]
Are there any plans to update Cucable to allow it to run on the latest version of Cucumber?  Or, would you be open to a PR that does that update?  Since Cucumber 5+ can run in parallel natively (and since there are breaking changes in Cucumber 6 that would probably require a bump to version 2.0.0 for Cucable), I understand if you prefer to just leave the project at Cucumber 5.  Still, I thought it was worth asking about.

**Comment by john-caleb-phillips** [comment]
Just to clarify, after double checking the code, what actually needs to happen is that tag.expressions.version has to be updated to 3.0.0 to work with Cucumber 6+.

**Comment by laxersaz** [comment]
We are using Cucumber 6.8.0 with the current Cucable version so it does support it. Are there any specific features that don't work?

**Comment by john-caleb-phillips** [comment]
That's weird.  I'm getting errors like "java.lang.IncompatibleClassChangeError: Expected static method 'io.cucumber.tagexpressions.Expression io.cucumber.tagexpressions.TagExpressionParser.parse(java.lang.String)".  Must be something messed up in my configuration then.  I'll dig into it.

**Comment by laxersaz** [comment]
If you find something that is related to Cucable, please let me know.

**Comment by john-caleb-phillips** [comment]
My apologies.  It appears that I had a couple dependencies that were conflicting, and that was the root cause.  I was able to get it working, and will close this issue.

**Comment by RPRAB02** [comment]
Hello John , 
Can you please let me know as tho what was the exact dependency or steps you followed to remove the above issue .I am also getting the same.

**Comment by john-caleb-phillips** [comment]
Good Morning,

Unfortunately, I'm don't thing I'm going to be able to be much help with this.  I've since moved onto another project, and can't remember the exact fix that I used.  If I remember correctly, the issue in my case had something to do with a circular dependency in my pom.  So while I was updating the version in the parent pom, it didn't propagate correctly around the dependency loop.  Hopefully your project does not have that issue, as I wouldn't wish circular dependencies on anyone.  Than again, the circular dependency was the cause of a lot of issues, so I might be conflating two things in my memory.

Thanks,
John Phillips
REI Systems ... Achieving Excellence Together
Mobile: 609.417.2400
***@***.******@***.***> | www.reisystems.com<http://www.reisystems.com/>
